card/lib/cardio.rb

```diff
@@ -54,9 +38,8 @@ def add_card_paths paths, config, root
 
       add_gem_path paths, 'gem-mod',             :with => 'mod'
       add_gem_path paths, "db"
-      add_gem_path paths, "db/migrate"
+      add_gem_path paths, 'db/migrate'
       add_gem_path paths, "db/migrate_core_cards"
-      add_gem_path paths, 'db/migrate_deck_cards', :with=>'db/migrate_cards'
```

Comment:
Not sure where this went, maybe to the deck config.  In any case we have to pay close attention to getting all of this right.  Running the core and structure migrations from the gem (card/cardio) and then the deck migrations.  The generated migration for adding card structure has to be taken into account as well as it will need to be versioned with the host application.